### DISCUSSION TITLE
Section 4.5: Writing style

### DISCUSSION THREAD

**Discussion by hamogu** [astropy/astropy-v5.0-paper#84] [Text comments] [answered]
> Now that the backend infrastructure is stable, we are interested in collecting new content
to serve and share: if you have ideas for contributions, or have tutorials hosted on
third-party sites that you would like to make searchable through the Learn Astropy
interface, please reach out by creating a GitHub issue in the Astropy Tutorials repos-
itory. 

I agree with the thought, but I want to point out that this style of writing is not what is usually employed in papers, nor does it match the rest of the paper. What if I read this paper 15 years from now, when we're all using Julia? Or after github brought up a paywall like RTD did? Also "please" is an usually worked for an article.

I'd cast it in a more descriptive style: Ideas or tutorial can be contributed by any interested party and will be hosted and ... The process to submit suggestion currently is to open a pull request on ...

**Comment by pllim** [comment]
I agree. Feel free to PR!

**Comment by adrn** [comment, marked as answer]
I addressed this in a commit that I will push soon. Thanks!
